bazelisk.go

```diff
@@ -140,7 +150,7 @@ func maybeDownload(bazeliskHome, url, filename, description string) ([]byte, err
 	}
 
 	// We could also use go-github here, but I can't get it to build with Bazel's rules_go and it pulls in a lot of dependencies.
-	body, err := readRemoteFile(url)
```

Comment:
By doing this I've taken over `maybeDownload`, I'm happy to rename it if that makes sense.